util/findLostPatchesOneRepository.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/sh
+
+###########################################
+#  
+#   Find any missing commits that are IN a given branch 
+#   but are NOT in the 2nd branch. 
+#  
+#   These missing commits are found via a concept
```

Comment:
"via a concept called patch-id" sounds a bit high-fallutin'. Why not just "via patch-id" ? Ya'll gettin' fancy?